.env.example

```diff
@@ -15,3 +19,5 @@ JWT_SECRET=secret
 
 WEB_FRONTEND_URL=https://keyshade.xyz
 WORKSPACE_FRONTEND_URL=https://app.keyshade.xyz
+
+NODE_ENV='development <prod | dev>'
```

Comment:
This one will introduce problems for us. There are other scripts that are using this variable via code. I suggest you pass this from the commands itself. Either ways, sentry should only be disabled when the profile is `e2e` or `test`.

Reply:
Got it!